src/components/rangeFilterInput/__tests__/Index.Test.tsx

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable testing-library/no-unnecessary-act */
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+import { act, render, screen } from '@testing-library/react';
+
+import RangeFilterInput from '../index';
+
+type ChangeCallback = (event: { value: number; name: 'from' | 'to' }) => void;
+
+describe('<RangeFilterInput/>', () => {
+  const renderInputField = (onChange?: ChangeCallback) => {
```

Comment:
You could go witn 
```suggestion
  const renderInputField = (onChange = jest.fn()) => {
```

Then you don't need the type and the ternary in line 15